web/sites/default/config/field.field.node.page.field_webform.yml

```diff
@@ -16,7 +16,7 @@ field_name: field_webform
 entity_type: node
 bundle: page
 label: Webform
-description: "If a Portland.gov webform was created for this service, select it from the list below to embed it directly into the service page. If you're unsure, contact a member of the Website Support team at webforms@portlandoregon.gov for assistance."
+description: "If a Portland.gov webform was created for use with this service or guide, select it from the list below to embed it directly into the page. Guides don't typically have webforms, but there are some cases where they are used for their conditional logic, to guide users through decision trees. If you're unsure, contact a member of the Website Support team at webforms@portlandoregon.gov for assistance."
```

Comment:
This allows the Guide page type to also have a webform, since some guides may employ webform conditional logic.